addons/stock/models/stock_orderpoint.py

```diff
@@ -332,8 +332,10 @@ def _get_orderpoint_action(self):
         to_refill = defaultdict(float)
         all_product_ids = []
         all_warehouse_ids = []
-        # Take 3 months since it's the max for the forecast report
-        to_date = add(fields.date.today(), months=3)
+        ConfigParameter = self.env['ir.config_parameter'].sudo()
+        # By default, take 3 months
+        forecast_interval_months = ConfigParameter.get_param("stock.forecast_interval_months", "3")
```

Comment:
```suggestion
        # By default, take 3 months
        forecast_interval_months = self.env['ir.config_parameter'].sudo().get_param("stock.forecast_interval_months", "3")
```
Avoid to declare variable when it's only used once. Otherwise okay